src/Microsoft.Health.Dicom.Core/Features/Store/StoreDatasetValidator.cs

```diff
@@ -284,11 +343,14 @@ private void ValidateWithoutNullPadding(string value, DicomElement de, IReadOnly
     }
 
     private static KeyValuePair<string, object>[] TelemetryDimension(DicomItem item, bool isIndexableTag) =>
+        TelemetryDimension(item.Tag, isIndexableTag);
+
+    private static KeyValuePair<string, object>[] TelemetryDimension(DicomTag tag, bool isIndexableTag) =>
         new[]
         {
-            new KeyValuePair<string, object>("TagKeyword", item.Tag.DictionaryEntry.Keyword),
-            new KeyValuePair<string, object>("VR", item.ValueRepresentation.ToString()),
-            new KeyValuePair<string, object>("Tag", item.Tag.ToString()),
+            new KeyValuePair<string, object>("TagKeyword", tag.DictionaryEntry.Keyword),
+            new KeyValuePair<string, object>("VR", tag.GetDefaultVR()),
```

Comment:
we are changing the type that we are storing here from string to VR... we were explictly not doing that before.